.github/CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax
+
+# The Auro team will be the default owners for everything in the repo.
+*       @AlaskaAirlines/auro-team
+
+# Order is important. The last matching pattern has the most precedence.
+# If a pull request touches any files in the ./src dir, only these owners
+# will be requested to review.
+./src/*    @AlaskaAirlines/auroreviewers
```

Comment:
@blackfalcon 

It looks like line 6 maps to: https://github.com/orgs/AlaskaAirlines/teams/auro-team

But who does line 11 map to?

Also, looks like `auroreviewers` should also have coverage of `./docs/*`, `./demo/*` and `./readme.md`

Reply:
`@AlaskaAirlines/auroreviewers` is the sub-team I spoke of that is just you and me. The declaration here is that if there are any changes in `./src` then you or I have to have eyes on that. 

Anything else like `./docs/*`, `./demo/*` and `./readme.md` will go to the whole `@AlaskaAirlines/auro-team` so we will have eyes on it, but not so directly. 

The idea here is growth. Say we add someone to the team, they will get a notice for anything, but again, when the real meat of a change happens in the `./src` dir, we will be notified until we decide to let the new person into the secret circle.